src/Send/SendScreen/utils.ts

```diff
@@ -127,8 +130,8 @@ export const recomputeAll = async ({
       // we'll substract minAda from ADA balance if we are sending a token
       const minAda =
         !selectedTokenInfo.isDefault && isHaskellShelleyNetwork(selectedTokenInfo.networkId)
-          ? new BigNumber(await getMinAda(selectedTokenInfo, defaultAsset))
-          : new BigNumber('0')
+          ? (new BigNumber(await getMinAda(selectedTokenInfo, defaultAsset)).toString() as Quantity)
+          : (new BigNumber('0').toString() as Quantity)
```

Comment:
you dont need to convert to  BigNumber if its already a string...

Reply:
Done. Thanks